### PR TITLE
Python Examples

### PR DESCRIPTION
**Pull request by pavankumarbn**
@ajpmaclean @lorensen Please review these examples.

### CODE REVIEW

File: src/Python/Geovis/GeoGraticle.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+def main():
+    colors = vtk.vtkNamedColors()
+    geoGraticle = vtk.vtkGeoGraticule()
+    transformProjection = vtk.vtkGeoTransform()
+    destinationProjection = vtk.vtkGeoProjection()
+    sourceProjection = vtk.vtkGeoProjection()
+    transformGraticle = vtk.vtkTransformFilter()
+    
+	  reader = vtk.vtkXMLPolyDataReader()
+    transformReader = vtk.vtkTransformFilter()
```
Comment (ajpmaclean) [inline src/Python/Geovis/GeoGraticle.py]: Please fix layout here.

File: src/Cxx/Geovis/GeoGraticle.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ### Description
 
-vtkGeoGraticle generates polydata to illustrate the distortions introduced by a map projection. The level parameter specifies the number of lines to be drawn. Poles are treated differently than other regions; hence the use of a Level parameter instead of a NumberOfLines parameter. The latitude and longitude are specified as half-open intervals with units of degrees. By default the latitude bounds are [-90,90[ and the longitude
-bounds are [0,180[.
+vtkGeoGraticle generates polydata to illustrate the distortions introduced by a map projection. The level parameter specifies the number of lines to be drawn. Poles are treated differently than other regions; hence the use of a Level parameter instead of a NumberOfLines parameter. The latitude and longitude are specified as half-open intervals with units of degrees. By default the latitude bounds are [-90,90] and the longitude
+bounds are [0,180].
```
Comment (ajpmaclean) [inline src/Cxx/Geovis/GeoGraticle.md]: Could you check this? Given that these are half-open intervals I think the they should be [-90...90) and [0...180) or maybe the open interval (-90...90) -  more likely. In looking at the code I think these should be (-90...90) and (-180...180) i.e. open intervals.

Comment (pavankumarbn) [reply src/Cxx/Geovis/GeoGraticle.md]: @ajpmaclean Sure I will change it.

**Comment by pavankumarbn** [comment]
@ajpmaclean I have made the changes. Please review on the same.

**Comment by ajpmaclean** [comment]
Thanks, looks good.

**Comment by pavankumarbn** [comment]
@ajpmaclean Thank you

**Pull request merged**